### ISSUE TITLE
Decide which cloud database we should use.

### ISSUE THREAD

**Issue by greenson** [nyu-devops-fall19-suppliers/suppliers#122]
**As a** developer
**I need** our app to work properly after it is migrated to IBM cloud
**So that** we have to choose a database system that IBM cloud supports.

**Assumptions:**
* The free IBM Cloudant (which was built on open source CouchDB, a NoSQL database like MongoDB). However our codes may be altered accordingly. 
https://cloud.ibm.com/catalog/services/cloudant

* The non-free MongoDB. And it should be compatible with our existing work.
https://cloud.ibm.com/catalog/services/databases-for-mongodb

**Acceptance Criteria**
```
Given either option we choose, after setting up the db in IBM Cloud
When we run nosetests
Then all test cases should be passed
```

**Comment by lhmartin** [comment]
I think we should  just use a free version, no need to pay for anything. Plus other teams are doing the same, and we can find out how they fixed their code to work with it.

**Comment by silemie** [comment]
I know the team that uses MongoDB changed to use CouchDB. But if using CouchDB, we should modify our code and vagrant. I'm not sure how many lines we should modify.

**Comment by silemie** [comment]
I found that the lab-flask-bdd is deployed in the IBM Cloud using CouchDB. That is helpful for us!

**Comment by greenson** [comment]
Another option: MongoDB is running its own cloud database
[https://www.mongodb.com/cloud/atlas](url)

And it seems free!

**Comment by silemie** [comment]
That looks good. I vote for using that.

**Comment by lhmartin** [comment]
Agreed!

**Comment by lhmartin** [comment]
It appears as if there is no connection to IBM cloud given as an option
<img width="906" alt="Screenshot 2019-11-19 at 18 28 21" src="https://user-images.githubusercontent.com/16001690/69195674-5e64fd80-0afa-11ea-80c4-290a0cf67e3e.png">

**Comment by lhmartin** [comment]
@greenson @silemie @Krystal-158 Since Atlas doesn't work, should we just try change to Couldant?

**Comment by greenson** [comment]
> @greenson @silemie @Krystal-158 Since Atlas doesn't work, should we just try change to Cloudant?

@lhmartin 
Sorry mate I didn't notice this issue was reopened 😂

**TLDR: Relax we are good** 😁 
As the assignment description says:
![image](https://user-images.githubusercontent.com/17645845/69206554-fd015680-0b1a-11ea-8752-0235858ede04.png)
so we are just using a 3rd party data like PostgreSQL.

The AWS or Azure stuff is merely the host of MongoDB Atlas. It does not limit any ability to connect to other cloud services that are not on the list. Actually we have been using MongoDB Atlas since this commit: https://github.com/nyu-devops-fall19-suppliers/suppliers/pull/127 and it works great 😁
